migrations/20161013215314-dropglobalconfig.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2016, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * Because a deployment may have a poorly-defined GlobalConfig table at this
+ * time, this migration tests for the existence of the "createdAt" field and
+ * only drops the table if that field does not exist. After this migration,
+ * when the server starts up and index.js calls sync on all the models, the
+ * GlobalConfig table will be created properly.
+ */
+'use strict';
+
+module.exports = {
+  up: function (queryInterface, Sequelize) {
+    return queryInterface.describeTable('GlobalConfig')
```

Comment:
I got this error when I do postinstall:
```
Using environment "development".
== 20161013215314-dropglobalconfig: migrating =======

stderr: No description found for "GlobalConfig" table. Check the table name and schema; remember, they _are_ case sensitive.
```
The error goes when I change the name of the table to "GlobalConfigs"